ckanext/gla/templates/about/accessibility.html

```diff
@@ -0,0 +1,48 @@
+{% extends "page.html" %}
+
+{% block main_content %}
+<div class="module-content">
+    <h1>Accessibility</h1>
+    <p>This website is run by the Greater London Authority (GLA). We want as many people as possible to be able to use this website. For example, that means you should be able to:</p>
+    <ul>
+    <li>change colours, contrast levels and fonts</li>
+    <li>zoom in up to 300% without the text spilling off the screen</li>
+    <li>navigate most of the website using just a keyboard</li>
+    <li>navigate most of the website using speech recognition software</li>
+    <li>listen to most of the website using a screen reader (including the most</li>
+    <li>recent versions of JAWS, NVDA and VoiceOver)</li>
```

Comment:
I don't think this should be a new `<li>` - it's one sentence:
![image](https://github.com/GreaterLondonAuthority/dfl-ckanext/assets/20282373/597c9e8d-1776-41cb-8cb5-75cbfdb13bf3)